Services/Mail/classes/class.ilMailOptions.php

```diff
@@ -1,6 +1,7 @@
 <?php
 /* Copyright (c) 1998-2012 ILIAS open source, Extended GPL, see docs/LICENSE */
 
+require_once 'Services/Mail/classes/Options/class.ilMailTransportSettings.php';
```

Comment:
With ILIAS 5.3.x we don't need require/include statements anymore. I suggest to remove this an re-build the class map according to ./libs/composer/README.md